core/storage/user/gae_models_test.py

```diff
@@ -84,6 +84,69 @@ def test_get_failure(self):
         self.assertEqual(retrieved_object, None)
 
 
+class UserStatsModelTest(test_utils.GenericTestBase):
+    """Tests for the UserStatsModel class."""
+
+    USER_ID_1 = 1
+    USER_ID_2 = 2
+
+    USER_1_IMPACT_SCORE = 0.87
+    USER_1_TOTAL_PLAYS = 33
+    USER_1_AVERAGE_RATINGS = 4.37
+    USER_1_NUM_RATINGS = 22
+    USER_1_WEEKLY_CREATOR_STATS_LIST = [
+        {
+            ('2019-05-21'): {
+                'average_ratings': 4.00,
+                'total_plays': 5
+            }
+        },
+        {
+            ('2019-05-28'): {
+                'average_ratings': 4.95,
+                'total_plays': 10
+            }
+        }
+    ]
+
+    def setUp(self):
+        """Set up user models in datastore for use in testing."""
+        super(UserStatsModelTest, self).setUp()
+
+        user_model_1 = user_models.UserStatsModel(id=self.USER_ID_1)
+        user_model_1.impact_score = self.USER_1_IMPACT_SCORE
+        user_model_1.total_plays = self.USER_1_TOTAL_PLAYS
+        user_model_1.average_ratings = self.USER_1_AVERAGE_RATINGS
+        user_model_1.num_ratings = self.USER_1_NUM_RATINGS
+        user_model_1.weekly_creator_stats_list = (
+            self.USER_1_WEEKLY_CREATOR_STATS_LIST)
+        user_models.UserStatsModel.put(user_model_1)
+
+    def test_export_data_on_existing_user(self):
```

Comment:
Maybe also add a multi-user test, e.g. that if you retrieve the data for a different user that it's the data for that user rather than the first user.

Reply:
addressed in latest commit